crates/server/src/admin/storage/jwt_token.rs

```diff
@@ -0,0 +1,64 @@
+use calimero_store::entry::{Entry, Json};
+use calimero_store::key::Generic;
+use calimero_store::Store;
+use serde::{Deserialize, Serialize};
+
+struct JwtRefreshTokenEntry {
+    key: Generic,
+}
+
+impl Entry for JwtRefreshTokenEntry {
+    type Key = Generic;
+    type Codec = Json;
+    type DataType<'a> = JwtRefreshToken;
+
+    fn key(&self) -> &Self::Key {
+        &self.key
+    }
+}
+
+impl JwtRefreshTokenEntry {
+    fn new() -> Self {
+        Self {
+            key: Generic::new(*b"jwt_token:server", [0; 32]),
+        }
+    }
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct JwtRefreshToken {
+    refresh_token: Vec<u8>,
```

Comment:
do we need to store the refresh token at all? we just need to ensure;

1. we can decode it from the user's request (so it was issued by us)
2. it has not expired

right?

Reply:
Access token is short valid token. Refresh token is used to obtain new access token

Reply:
Yeah, but do we need to store it?

Reply:
You can use it to block some sessions. e.g. remove access from external location by deleting refresh key. Access token will be valid until expires but new one will not be issued